README.md

```diff
@@ -1895,6 +1895,7 @@ _Related: [Monitoring](#monitoring)_
 - [s.Status](https://github.com/scolastico-dev/s.Status) - Open-source server status page written in java. `MPL-2.0` `Java`
 - [StatPing.ng](https://statping-ng.github.io/) - An easy to use Status Page for your websites and applications. Statping will automatically fetch the application and render a beautiful status page with tons of features for you to build an even better status page. ([Source Code](https://github.com/statping-ng/statping-ng)) `GPL-3.0` `Docker/Go`
 - [Uptime Kuma](https://github.com/louislam/uptime-kuma) - Self-hosted website monitoring tool like "Uptime Robot". ([Demo](https://demo.uptime.kuma.pet)) `MIT` `Nodejs`
+- [Vigil](https://crates.io/crates/vigil-server) - Microservices Status Page. Monitors a distributed infrastructure and sends alerts (Slack, SMS, etc.). ([Source Code](https://github.com/valeriansaliou/vigil)) `MPL-2.0` `Rust/Docker`
```

Comment:
https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3269 is a duplicate of this pull request, I will merge 3269 instead of this one since it also adds a demo link and remove the now useless redirect to awesome-sysadmin.